packages/arb-token-bridge-ui/src/components/App/App.tsx

```diff
@@ -170,11 +170,13 @@ const Injector = ({ children }: { children: React.ReactNode }): JSX.Element => {
     // Any time one of those changes
     setTokenBridgeParams(null)
     actions.app.setConnectionState(ConnectionState.LOADING)
+
     if (!isConnected || !chain) {
       return
     }
 
-    const { l1, l2, isConnectedToArbitrum } = networksAndSigners
+    const { l1, l2 } = networksAndSigners
+    const isConnectedToArbitrum = isNetwork(chain.id).isArbitrum
```

Comment:
Don't we want to just utilize the hook here?

Reply:
This seemed a bit cleaner, because otherwise I'd have to call the hook at the top of the component, and add another dependency to the `useEffect`, which can cause some unnecessary re-renders. In addition, I'm pretty sure we'll be able to kill that `useEffect` with https://github.com/OffchainLabs/arbitrum-token-bridge/issues/852.